README.md

```diff
@@ -704,6 +705,10 @@ $ php vendor/bin/phpunit
 
 MIT, see [LICENSE file](LICENSE).
 
+## Known Issues
+* _*pecl-event*_ on Windows is not capable of accepting arbitrary file descriptor types (`EventBaseConfig::FEATURE_FDS`).
```

Comment:
The class is called `EventConfig`, not `EventBaseConfig` (that's only the alias in the event loop implementation).

Reply:
fixed